release-notes/2023-11/README.md

```diff
@@ -1,31 +1,37 @@
 # SystemLink Enterprise release 2023-11 Release Notes
 
-The 2023-11 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+The 2023-11 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
 
 ## New Features and Behavior changes
 
-- Behavior change or new feature description
-
-- workorderservice
-    - The following privilege is added in the workorder to enable the "schedule" access to the test plan: `testplan:Schedule`.
-    - Note that the workorder service is not added in the helm chart. Hence, both workorder and testplan will not be accessible in the application.
+- Work Orders
+    - The Work Order's expose a new privilege for scheduling test plans.
+    - The Work Order service and UI is not included in the SystemLink Helm chart and associated privileges can be ignored.
 
 ## Helm Chart Breaking Changes
 
-- sl-jupyterhub
-    - Notebooks are now running on Python 3.11.
+- `sl-jupyterhub`
+    - Notebooks now support Python 3.11 in the JupyterHub development environment.
+
+- `nbexecservice`
+    - Notebooks now support Python 3.11 in notebooks executed by Routines.
+
+- `testmonitorservice 0.16.37`
+    - The update includes a new database schema migration to support future features. This migration adds a new column to the `results` table. The migration is expected to complete without downtime. Downgrading to prior versions of the TestMonitor service after updating to this version is not supported.
```

Comment:
I include "This migration adds a new column to the `results` table." in the excel sheet, but I don't think customers really need to know that. In our other documentation we say we will change the schema without notice on what we are changing. Up to you whether we leave it or not.

Reply:
Ok, I was on the fence about including this. I will drop. Thanks for the confirmation.